src/scripts/main.js

```diff
@@ -1,3 +1,10 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
```

Comment:
Ensure that the element with the class 'wall' exists in the DOM before accessing its properties. If it doesn't exist, `wall` will be `null`, and accessing `offsetHeight` or `offsetWidth` will result in an error.